.github/workflows/docker.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image
+on:
+  push:
+    tags:
```

Comment:
Seems redundant? when we're already running on push to main

Reply:
We need as well catch `tags` to create build when you will make new release
<img width="895" alt="Zrzut ekranu 2023-04-12 o 09 24 16" src="https://user-images.githubusercontent.com/6894763/231382848-2c9f6ea2-9496-44d6-b532-ecdb7792d1ba.png">

Reply:
ah is it like for tags on main branch?

Reply:
<img width="478" alt="Zrzut ekranu 2023-04-12 o 09 27 10" src="https://user-images.githubusercontent.com/6894763/231383613-d1cd8009-af2c-47eb-8bfb-c7170205b9c0.png">
it's for above tags

Reply:
Ok so that makes more sense if it's only for tags and not for every commit

Reply:
yeah, it's for tags

Reply:
Alright